WpfSampleDi/WpfSampleDi.csproj

```diff
@@ -1,13 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
-    <OutputType>WinExe</OutputType>
+    <OutputType>Exe</OutputType>

```

Comment:
This is just to demonstrate how Console Logger works (otherwise console window is not created and you can't see the logs).